### DISCUSSION TITLE
clob process variables

### DISCUSSION THREAD

**Discussion by bsdgithub** [flowsforapex/apex-flowsforapex#720] [Ideas]
do you consider that for clob data type (process variables) to introduce expression type like expression, sql query or procedure?

**Comment by rallen2010** [comment]
We suspect that many CLOB process variables today are being used to store json objects - and we are introducing a JSON data type process variable in the next release (24.1) which will make working with json much easier.  We can look at doing more with both json and club proc vars thereafter....

**Reply by bsdgithub** [reply]
that's true, vast majority will be json or xml, if json is on the way, I believe that will cover much of the cases.
